apps/src/p5lab/spritelab/poembot/commands/backgroundEffects.js

```diff
@@ -8,6 +8,21 @@ export const commands = {
       this.p5.background(color);
     };
   },
+
+  setBackgroundImageAs(imageName) {
+    if (
+      this.p5._predefinedSpriteAnimations &&
+      this.p5._predefinedSpriteAnimations[imageName]
+    ) {
+      let backgroundImage = this.p5._predefinedSpriteAnimations[imageName];
+      backgroundImage.name = imageName;
+      backgroundImage.resize(400, 400);
```

Comment:
i know the value 400 is scattered all over the place in our labs, but we should move toward using the p5lab constants in case we ever want to change this value

https://github.com/code-dot-org/code-dot-org/blob/f26d5d6e6327d3a864d7c7062394885889f76a6e/apps/src/p5lab/constants.js#L7-L11